### PR TITLE
fix shared-search to index multiple files from the same work

### PR DESCRIPTION
**Pull request by mankind**
https://trello.com/c/X4BlQc1v/562-15916-not-working-reindex-a-work-that-is-already-in-shared-search-in-certain-scenarios

### CODE REVIEW

File: app/models/file_set.rb [hunk 1]
```diff
@@ -14,12 +19,20 @@ def rendering_ids
...
-  def account_cname
+  def _account_cname
     parent.try(:account_cname)
   end
 
   private
 
+    def set_account_cname
+      if self.account_cname.blank?
+        object ||=  Account.where("tenant ilike ?", "%#{Apartment::Tenant.current}%").where("data @> ?", {is_parent: 'false'}.to_json ).first
```
Comment (ashikajith) [inline app/models/file_set.rb]: @mankind   `Account.where("tenant **ilike** ?"` -> Typo ?

**Comment by mankind** [comment]
Not typo. Run it in rails console to see for yourself. Using ilike means it
is case insensitive.

File: app/models/file_set.rb (hunk 1 above)
Comment (mankind) [reply app/models/file_set.rb]: Not a typo. Run it in rails console to see for yourself. Using ilike means it is case insensitive.

Comment (ashikajith) [reply app/models/file_set.rb]: +1

**Pull request merged**